src/build.ts

```diff
@@ -359,6 +359,7 @@ async function _build(nitro: Nitro, rollupConfig: RollupConfig) {
   const buildInfo = {
     date: new Date(),
     preset: nitro.options.preset,
+    entry: nitro.options.entry.split("/").pop(),
```

Comment:
We have no way to know which entry a preset use, maybe this should be in another PR

Reply:
Extracted in https://github.com/unjs/nitro/pull/1649